### DISCUSSION TITLE
Merge Queue Beta resulting in duplicate merged commits

### DISCUSSION THREAD

**Discussion by jaredlockhart** [community/community#47458] [Pull Requests]
### Select Topic Area

Bug

### Body

I just enabled Github Merge Queue Beta on my repo here:

https://github.com/mozilla/experimenter

with the default settings here

<img width="683" alt="image" src="https://user-images.githubusercontent.com/119884/218835188-078c417e-a1cd-4251-a92d-cf722fe567c6.png">

It seemed to work okay for the first few merged PRs, but now there is a PR in the queue

<img width="1249" alt="image" src="https://user-images.githubusercontent.com/119884/218833509-6043a241-2b59-4249-bca1-025614c9f2e5.png">

and it seems to keep restarting its CI checks 

<img width="1296" alt="image" src="https://user-images.githubusercontent.com/119884/218833614-d29755af-e35f-4cb4-b9b9-c6c31104c171.png">

and every time it completes it adds another duplicate commit to main

<img width="1232" alt="image" src="https://user-images.githubusercontent.com/119884/218833668-a566d109-abee-4110-8148-5a2ca7f89e97.png">

Is this a bug with merge queues or have I configured something incorrectly?

Thank you.

**Comment by ziluvatar** [comment]
In my case, when a commit is duplicated, one shows checks from main and the other one shows checks from github merge group branch.

**Comment by eliihen** [comment]
We also experienced this today, see below. Merge queue worked fine for a while, but it went amok today when the below PR was merged at Mar 10, 2023, 10:35 AM GMT+1. We used default settings as well.

![image](https://user-images.githubusercontent.com/5845924/224329210-1c89cbed-0455-4e99-8cfc-9681bd5ed890.png)
